File: app/src/main/java/app/rental/utsavstha/rentalapp/Utils/ShowMessage.java
package app.rental.utsavstha.rentalapp.Utils;

import android.widget.Toast;

import app.rental.utsavstha.rentalapp.RentApplication;

/**
 * Created by utsavstha on 8/19/17.
 */

public class ShowMessage {
    public static void toast(String message){
        Toast.makeText(RentApplication.getContext(), message, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/app/rental/utsavstha/rentalapp/Utils/ShowMessage.kt
package app.rental.utsavstha.rentalapp.Utils

import android.widget.Toast

import app.rental.utsavstha.rentalapp.RentApplication

/**
 * Created by utsavstha on 8/19/17.
 */

object ShowMessage {
    fun toast(message: String, duration: Int = Toast.LENGTH_SHORT) {
        Toast.makeText(RentApplication.context, message, duration).show()
    }
}
